skimage/transform/tests/test_geometric.py

```diff
@@ -288,6 +321,45 @@ def test_affine_shear():
     assert_almost_equal(tform.params, expected)
 
 
+@pytest.mark.parametrize(
+    'pts,params',
```

Comment:
```suggestion
    'pts, params',
```
Unless this breaks the parsing? I haven't actually checked!